docs/dev/annual_updates.rst

```diff
@@ -18,39 +20,46 @@ This document outlines all the tasks required to complete the annual update, bas
 our experience in 2021. You can look at :issue:`1255` to see all the commits that went
 into integrating the 2020 data.
 
-Obtain fresh data
------------------
-Scrape a new copy of the raw PUDL inputs from agency websites using the tools in the
+1. Obtain Fresh Data
+--------------------
+**1.1)** Scrape a new copy of the raw PUDL inputs from agency websites using the tools
+in the
 `pudl-scrapers repository <https://github.com/catalyst-cooperative/pudl-scrapers>`__.
 If the structure of the web pages or the URLs have changed, you may need to update the
 scrapers themselves.
 
-Use the newly scraped files and the tools in the `pudl-zenodo-storage repository
-<https://github.com/catalyst-cooperative/pudl-zenodo-storage>`__ to create new raw input
-achives on Zenodo. If we've previously archived the dataset this will add a new version
-to an existing archive.
+**1.2)** Use the newly scraped files and the tools in the `pudl-zenodo-storage
+repository <https://github.com/catalyst-cooperative/pudl-zenodo-storage>`__ to create
+new raw input achives on Zenodo. If we've previously archived the dataset this will add
+a new version to an existing archive.
 
 .. note::
-
     It is useful to run the ``zenodo_store.py`` script using the ``--noop`` option first
-    to see what actions will be taken. Take note of any older years of data that have
-    been changed, so that you can keep an eye out for those changes later in the data
-    integration process.
+    to see what actions will be taken. In addition to the new year, **take note of any
+    older years of data that have been retroactively changed** so that you can address
+    them later on in the file mapping and transform updates.
+
+**1.3)** Update the dictionary of production DOIs in :mod:`pudl.workspace.datastore` to
+refer to the new raw input archives.
 
-Update the dictionary of production DOIs in :mod:`pudl.workspace.datastore` to refer to
-the new raw input archives and update the :py:const:`pudl.constants.WORKING_PARTITIONS`
-dictionary to reflect the data that is now available within each dataset.
+**1.4)** Update the working partitions in the :mod:`pudl.metadata.sources` dictionary to
+reflect the years of data that are available within each dataset.
+
+**1.5)** Update the years of data to be processed in the ``etl_full.yml`` and
+``etl_fast.yml`` settings files stored under ``src/pudl/package_data/settings`` in the
+PUDL repo.  Note that you will also need to have your own working copies of the settings
```

Comment:
you can rerun `pudl_setup {path to your pudl_work directory} -c` to update your settings files

Reply:
That's only possible once you update the working partitions, right?

Reply:
yes exactly!